packages/shared/src/modules/users/libs/enums/user-exception-message.enum.ts

```diff
@@ -2,6 +2,7 @@ const UserExceptionMessage = {
 	INVALID_CREDENTIALS: "Invalid credentials",
 	USER_ALLREADY_EXISTS: "A user with this email already exists",
 	USER_NOT_FOUND: "User not found",
+	USER_PROFILE_NOT_PUBLIC: "User profile are hidden by user",
```

Comment:
```suggestion
	PROFILE_NOT_PUBLIC: "User profile are hidden by user",
```
see: https://github.com/BinaryStudioAcademy/bsa-2025-smartscapes/pull/451#discussion_r2287795016